### PR TITLE
Update Arriba ResourceRequirement

### PR DESCRIPTION
**Pull request by skanwal**
Resolves https://github.com/umccr/cwl-ica/issues/333.

### CODE REVIEW

File: tools/arriba-fusion-calling/2.4.0/arriba-fusion-calling__2.4.0.cwl [hunk 1]
```diff
@@ -26,7 +26,7 @@ hints:
   ResourceRequirement:
     ilmn-tes:resources/tier: standard
     ilmn-tes:resources/type: standard
-    ilmn-tes:resources/size: xlarge
+    ilmn-tes:resources/size: xxlarge
     coresMin: 4
     ramMin: 14000
```
Comment (alexiswl) [inline tools/arriba-fusion-calling/2.4.0/arriba-fusion-calling__2.4.0.cwl:31]: Should now be

```yaml
coresMin: 8
ramMin: 28000
```

As per https://illumina.gitbook.io/ica-v1/analysis/a-taskexecution

File: tools/arriba-drawing/2.4.0/arriba-drawing__2.4.0.cwl [hunk 2]
```diff
@@ -26,7 +26,7 @@ hints:
   ResourceRequirement:
     ilmn-tes:resources/tier: standard
     ilmn-tes:resources/type: standard
-    ilmn-tes:resources/size: xlarge
+    ilmn-tes:resources/size: xxlarge
     coresMin: 4
     ramMin: 14000
```
Comment (alexiswl) [inline tools/arriba-drawing/2.4.0/arriba-drawing__2.4.0.cwl:31]: Should now be

```yaml
coresMin: 8
ramMin: 28000
```

As per https://illumina.gitbook.io/ica-v1/analysis/a-taskexecution

Comment (skanwal) [reply tools/arriba-drawing/2.4.0/arriba-drawing__2.4.0.cwl]: updated in https://github.com/umccr/cwl-ica/commit/7c0f499d20492d39214be99c2d5055b36d9e365b

File: tools/arriba-fusion-calling/2.4.0/arriba-fusion-calling__2.4.0.cwl (hunk 1 above)
Comment (skanwal) [reply tools/arriba-fusion-calling/2.4.0/arriba-fusion-calling__2.4.0.cwl]: updated in https://github.com/umccr/cwl-ica/commit/7c0f499d20492d39214be99c2d5055b36d9e365b

**Comment by alexiswl** [comment]
Waiting for #391 to finish

**Review by alexiswl — APPROVED** [review]

**Pull request merged**